### ISSUE TITLE
relay-compiler 13.0.1 writes to `stderr`

### ISSUE THREAD

**Issue by hanford** [facebook/relay#3743]
The new compiler seemingly writes useful information to `stderr` which can be problematic as some CI environments will error if `stderr` is written to

This can be verified by using a redirect operator and redirecting `stderr` to a file

```
relay-compiler &> foo
```

.. if this is intentional, feel free to close

https://linuxize.com/post/bash-redirect-stderr-stdout/#redirecting-stderr-to-stdout

**Comment by ch1ffa** [comment]
By default, env_logger logs to stderr. It can be changed by `target` option, but errors will be redirected to stdout either. So we need to use another logger to split to stdout/err, for example `simplelog`.

I've tried simplelog::TermLogger and it looks good. Just added
```rust
let config = ConfigBuilder::new()
    .set_time_level(LevelFilter::Off)
    .build();

TermLogger::init(
    LevelFilter::Info,
    config,
    TerminalMode::Mixed,
    ColorChoice::Auto,
).unwrap();
```
instead of env_logger, and output was quite similar to the previous one and split to stdout/err:
<img width="587" alt="Screenshot 2022-01-17 at 00 26 00" src="https://user-images.githubusercontent.com/17417010/149678725-63fb7ad1-ff3b-4428-896d-70b98dc52859.png">

**Comment by alunyov** [comment]
This fix was added in #3763. Thanks @ch1ffa!

**Comment by hanford** [comment]
Thanks, feel free to close out whenever is appropriate @alunyov
